src/styles/diagnostic.css

```
@@ -117,18 +123,61 @@ h3 {
+  pointer-events: none;
+}
+
 .wb-device-label {
   font-size: 0.8rem;
   color: #888;
   margin-bottom: 0.5rem;
 }
 
+.wb-lane-health,
+.wb-timestamp-readout {
+  font-family: ui-monospace, "SFMono-Regular", Consolas, monospace;
```

Stylelint (17.7.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)